packages/api/src/external/commonwell/document/document-query.ts

```diff
@@ -725,13 +728,15 @@ async function triggerDownloadDocument({
   organization,
   facilityNPI,
   cxId,
+  requestId,
 }: {
   doc: DocumentWithLocation;
   fileInfo: S3Info;
   organization: Organization;
   facilityNPI: string;
   cxId: string;
-}): Promise<File> {
+  requestId: string;
+}): Promise<File | void> {
```

Comment:
I suggest we only use `void` when we don't care what the function returns, as the docs/spec states. In this case we want to know whether it returned File or didn't return anything (`undefined`).